linux/keyman-config/tests/test_bcp47tag.py

```diff
@@ -82,6 +82,14 @@ def test_SetTag(self):
         # Verify
         self.assertEqual(Bcp47Tag('sr', 'Cyrl'), sut)
 
+    def test_SetTag_Update(self):
+        # Setup
+        sut = Bcp47Tag.create('glr-Latn-LR')
+        # Execute
+        sut.tag = 'glr'
```

Comment:
I'm not entirely sure what this is testing? How does it differ from the previous test?

Reply:
You're right, that's confusing. I'll change it. It's testing that setting a new tag will clear out the unspecified parts.